### PR TITLE
added webhook controller

### PR DESCRIPTION
**Pull request by aqib-git**

### CODE REVIEW

File: src/DocusignDriverServiceProvider.php [hunk 1]
```diff
@@ -16,6 +17,10 @@ public function boot(): void
         $this->publishes([
             __DIR__.'/../config/docusigndriver.php' => config_path('docusigndriver.php'),
         ]);
+
```
Comment (alnutile) [inline src/DocusignDriverServiceProvider.php]: CleanShot 2024-02-14 at 07 13 24@2x

Comment (alnutile) [reply src/DocusignDriverServiceProvider.php]: If there is a problem doing it the way they suggest let me know

File: src/Http/Controllers/ElectronicSignatureWebhookController.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public function handleWebhook(Request $request): JsonResponse
+    {
+        $webhook = $request->all();
+
+        if ($webhook['event'] !== 'recipient-completed') {
+            return new JsonResponse([], 200);
+        }
+
+        \Log::info($webhook);
```
Comment (alnutile) [inline src/Http/Controllers/ElectronicSignatureWebhookController.php:21]: ok so make a Job in the next PR
and save the payload to the folder tests/fixtures/webhook.json so I can see what the incoming payload we want looks like so later I can write code and tests

**Review by alnutile — CHANGES_REQUESTED** [review]
one change please and then a request for the next PR

File: src/DocusignDriverServiceProvider.php (hunk 1 above)
Comment (aqib-git) [reply src/DocusignDriverServiceProvider.php]: done

**Pull request merged**